model/Asset.ts

```diff
@@ -2,6 +2,8 @@ type AssetInit = {
   id: string
   name: string
   description?: string
+  approved?: boolean
```

Comment:
Is this not mandatory in collibra?

Reply:
the status field is, but the field will always have a value regardless of what the status is, so i made it mandatory :+1: